### DISCUSSION TITLE
About push lsposed.zip to WSA

### DISCUSSION THREAD

**Discussion by Seasickil-CN** [LSPosed/LSPosed#1353] [General]
![mmexport1635861829229.png](https://user-images.githubusercontent.com/77569430/139864402-56b5588a-c73b-4f39-a454-a9c22fbf3d70.png)Ah..， I watched your teaching video, but I'm still not very good at it. Maybe I'm too stupid. Can you help me solve it?

**Comment by yujincheng08** [comment]
space.

**Reply by Seasickil-CN** [reply]
Thanks to the author, you helped me solve the problem!🥰
